pkg/clusterapi/apibuilder_test.go

```diff
@@ -397,21 +299,6 @@ func TestKubeadmConfigTemplate(t *testing.T) {
 	tt.Expect(got).To(Equal(want))
 }
 
-func TestKubeadmConfigTemplateWithRegistryMirror(t *testing.T) {
```

Comment:
Aren't these tests still valid?

Reply:
yea it's. i just moved the test to https://github.com/aws/eks-anywhere/pull/1916/files#diff-8a3a453f3e099d62cdc49e622592c91d0d7ec022e0e0c0cb8f1a87a58c2a42f7R112

Reply:
added back the apibuilder registry mirror tests